Add talks page with slides links

flexbox opened the pull request:
First draft, we need translation & some content for this

flexbox committed:
Add talks page with slides links

bobmaerten commented:
Why not add a `github` field to talks.yml entries ? Should be easy to fetch profile photo from here.

flexbox commented:
@cveneziani You can hack this here 
https://api.github.com/users/hackeryou

cveneziani committed:
Update talks page

- Add level to talks
- Add speakers details
- Add suggestion section
- Add translations for talks page
- Group talks by date

cveneziani commented:
Update:
- Add level to talks
- Add speakers details
- Add suggestion section
- Add translations for talks page
- Group talks by date

For now I'm not convinced about adding speaker picture. @flexbox can you propose something?

cveneziani committed:
Add all past talks

kdisneur commented on locales/en.yml:
```diff
@@ -70,3 +70,22 @@ en:
+      Find all the past talks from our Apéros Ruby.
+      Whether you're a beginner or a confirmed developer, feel free to give a talk and share your experience with the community.
+    level:
+      all: all levels
```
What's the difference between `all` and `soft`? If we want 4 levels I would suggest an `intermediate` instead.

cveneziani replied:
- `all` stands for any level
- `soft` stands for soft skill aka not a tech talk

What about using `soft skill`? And what would be the :fr: translation?
Sometimes French conferences don't translate it.

kdisneur replied:
OK make sense, but I have no idea how to translate that in french.

Moreover, what's the difference between `all` and `beginner`? I think I don't understand something.

For example:  Why `Présentation de la gem administrate` is `all` whereas `Craft everything with Middleman` is `beginner`?

cveneziani replied:
I'd say that `beginner` is for fundamental concepts / things that we're supposed to know, such as: 
- `What's the difference between class and instance variables`
- `Initiation to rails`

And that `all` is a new concept/gem that doesn't require a high level of expertise to get it (eg. administrate, chartkick, etc.).

Finally, for the middleman talk, I'd say it can be `all`. This is quite subjective :/
What do you think?

kdisneur replied:
I wouldn't use `all` to be honest. I would just use `beginner` because it works whatever your background is.
I really think the difference between `all` and `beginner` depends only of your background.

Example: I'm working with Ruby, mostly in a web environment. I never used `RubyMotion` in this web environment. An introduction to `RubyMotion` would be `all` for me whereas it would be `beginner` for someone who is used to do mobile development in ruby.

If you introduce a gem/subject and anyone can understand it, I think it should be `beginner` (even if confirmed ruby users might not know the gem/subject). 

I really think the level should be there just to specify the minimum skills you need to have to understand a talk.

`soft`: no ruby knowledge
`beginner`: small ruby knowledge 
`confirmed`: a bit more ruby knowledge

What do you think? Some others have feelings about this maybe?

cveneziani replied:
Makes sense. Agreed :)

kdisneur replied:
Actually we should probably do the inverse 😸 : use `all` instead of `beginner`.

I think that, as a confirmed user, you are more likely to look at slides tagged `all levels` than `beginner`.

tgautier replied:
How about using `any` instead of `all` if the goal of the term is to be a _catch-all_ for everyone?

cveneziani replied:
@kdisneur yep, that's why I started to introduce `all`
@tgautier :+1:  fol `any`

Ok so, do we agree to only keep `soft`, `any` and `confirmed`?

kdisneur commented on data/speakers.yml:
```diff
@@ -0,0 +1,48 @@
+douglas:
+  name: Douglas Camata
+  website: http://douglascamata.net/
+keuvain:
```
ahah 👍

cveneziani replied:
:grin:

flexbox committed:
Add swag for talks list

flexbox commented:
Style done!

cveneziani commented:
So nice!

![alignment-hysteria](https://cloud.githubusercontent.com/assets/50518/14762823/112163a8-0985-11e6-834f-230f6f72cdcd.jpg)

@flexbox Could you also align the speaker details with the title when this one is too long?

cveneziani committed:
Keep only 3 levels for talks: any, confirmed, soft

cveneziani committed:
Add link to Administrate gem on Thomas' talk

cveneziani commented:
![alignment-hysteria-first-letter](https://cloud.githubusercontent.com/assets/50518/14762969/2fe399f2-0988-11e6-8291-ea89aaa45f69.jpg)

@flexbox Can you also align the first letter? :)

cveneziani committed:
Fix title/url of talk about deployment

cveneziani committed:
Add link to talks page on home page

cveneziani commented:
Done for my part.
- Fixed levels + some titles/links of talks
- Added link to talks on home page

flexbox committed:
[gem] add middleman-sitemap-ping

flexbox committed:
fix alignment for avatar-card

flexbox commented:
Fix done ! I can proceed to merge :rocket:
